ProjectSidewalk/SidewalkWebpage
# Resolves #456 by adding share button and endpoint to save share images

Resolves #456

Added a share button and popup to the Gallery modal and created an endpoint on the server to save images for sharing on server.

##### Before/After screenshots
![image](https://user-images.githubusercontent.com/39865166/185003338-a59d0eba-7790-4dd6-b999-e344d5e2219d.png)
![image](https://user-images.githubusercontent.com/39865166/185003178-e2980f40-2d0b-4cbe-8271-1d8467873080.png)

##### Testing instructions
1.  Go to gallery and click on share button
2. Chose location to share and click on specific button
3. Check if title and image and everything else matches what you think it should be
4. (optional) check public/assets/share-images to make sure image is saved
Note: Needs to be on test server to properly check if image is being shown in preview.
